cmake/CMakeLists.txt

```diff
@@ -540,12 +541,14 @@ if(PKG_USER-PLUMED)
     ExternalProject_get_property(plumed_build INSTALL_DIR)
     set(PLUMED_INSTALL_DIR ${INSTALL_DIR})
     list(APPEND LAMMPS_DEPS plumed_build)
-    list(APPEND LAMMPS_LINK_LIBS ${PLUMED_INSTALL_DIR}/lib/libplumedWrapper.a ${CMAKE_DL_LIBS})
+    list(APPEND LAMMPS_LINK_LIBS ${PLUMED_INSTALL_DIR}/lib/plumed/obj/kernel.o
```

Comment:
this is different for plumed 2.5

Reply:
In CMake, the version is hard-coded, so we can fix it, when we bump it.

As said above due to https://github.com/plumed/plumed2/issues/402 I cannot even build plumed-2.5 at this point in time.

Reply:
@junghans you can just merge [the fix](https://github.com/plumed/plumed2/commit/dd7073f6fa0b64340c0a14c916ea0fb0214140b4) or configure plumed without python (`--disable-python`).

Reply:
Let's just bump the version to 2.5 in CMake, when you guys released it.

Reply:
@junghans following our usual schedule v2.5 is planned for (approx) December.

Reply:
If you guys prefer the beta version of 2.5 instead of 2.4.3 to be part of CMake we could bump it now.